laravel-directory-sync-app/resources/views/groups.blade.php

```diff
@@ -48,6 +67,7 @@
             </div>
         </div>
         <h2>You don't have any groups yet for this directory</h2>
```

Comment:
This part looks weird when there are no groups. Could we adjust the width so the text isn't scrunched, and then also get rid of the big WorkOS logo? Same thing on the Users page as well. 
![CleanShot 2022-08-26 at 14 26 55](https://user-images.githubusercontent.com/47123183/186985782-639e6764-0bce-48e9-a1ae-af7a286fa656.png)

Reply:
done